core/templates/dev/head/pages/exploration-editor-page/services/learner-answer-details-data.service.spec.ts

```diff
@@ -86,36 +75,104 @@ describe('Learner answer details service', function() {
     $httpBackend.verifyNoOutstandingRequest();
   });
 
-  it('should successfully fetch learner answer info data from the backend',
+  describe('.fetchLearnerAnswerInfoData', function() {
```

Comment:
This isn't very descriptive, ideally, describe descriptions start with `when...`

Reply:
Done